flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPool.java

```diff
@@ -163,4 +164,12 @@ void start(
 		@Nonnull SlotRequestId slotRequestId,
 		@Nonnull ResourceProfile resourceProfile,
 		@RpcTimeout Time timeout);
+
+	/**
+	 * Create report of the allocated slots on the specified task manager.
```

Comment:
`Create report about the allocated slots belonging to the specified task manager`